src/BCEngineMCMC.cxx

```diff
@@ -3745,6 +3729,33 @@ void BCEngineMCMC::Statistics::Reset(bool reset_mode, bool reset_efficiency)
     }
 }
 
+// ---------------------------------------------------------
+std::vector<std::vector<double> > BCEngineMCMC::Getx() const
+{
+    std::vector<std::vector<double> > x;
```

Comment:
I think it's nicer to set `x` to the known size instead of using `push_back`

Reply:
I've removed this code.